cli/src/main/java/hudson/cli/HexDump.java

```diff
@@ -28,7 +28,7 @@ public static String toHex(byte[] buf, int start, int len) {
                     inText = false;
                 }
                 r.append("0x");
-                r.append(CODE.charAt((b>>4)&15));
+                r.append(CODE.charAt(b >> 4 & 15));
```

Comment:
Are we confident everyone reading this knows the operator precedence rules by heart? Because this looks like something that would be more difficult to read.

Reply:
Good point. This trips up Error Prone's [`OperatorPrecedence`](https://errorprone.info/bugpattern/OperatorPrecedence) check as well. I'll rework this to retain the grouping parentheses for cases like this.